client/embedded-body-layout/embedded-body-props.ts

```diff
@@ -1,5 +1,6 @@
 export type EmbeddedBodyProps = {
 	page: string;
 	tab: string;
+	zone_id?: string;
```

Comment:
This feels very odd to me.  I wonder if there's a way to make this list more extensible so all components don't have to add in required params here.

Reply:
Yeah - I guess this _could_ be super generic by doing something like: `[key: string]: string;` 🤷

Reply:
That may work as well.  I was thinking along the lines of a filter, but not sure if JS hooks are completely obsolete or only where SlotFill is more appropriate.  cc @samueljseay 

Not a blocker for this PR, but I would like to get an issue created for follow-up if we don't handle here.

Reply:
Sure thing, I drafted something here: https://github.com/woocommerce/woocommerce-admin/issues/7501